### PR TITLE
Add beginnings of a CI, for now only pep8 and lint checks

### PR DESCRIPTION
**Pull request by grote**

### CODE REVIEW

**Comment by Skippern** [comment]
Maybe you should add the Travis status logo to `README.md`?

**Comment by grote** [comment]
Good point! Done!

**Comment by grote** [comment]
@nlehuby I had to change to some your just merged code to make the new CI pass. Please have a look if the changes are ok.

File: creators/accra/trips_creator_accra.py [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-from core.osm_routes import Route, RouteMaster
 
 
 class TripsCreatorAccra(TripsCreator):
+    ValueError = None
+    service_weekday = None
 
     def add_trips_to_schedule(self, schedule, data):
-        self.service_weekday = schedule.GetDefaultServicePeriod()
+        self.ValueError = schedule.GetDefaultServicePeriod()
```
Comment (nlehuby) [inline creators/accra/trips_creator_accra.py]: WTF ?

File: creators/accra/trips_creator_accra.py [hunk 2]
```diff
@@ -57,7 +56,7 @@ def add_trips_to_schedule(self, schedule, data):
                     if not ROUTE_FREQUENCY > 0:
                         print("frequency is invalid for route_master " + str(line.id))
                         ROUTE_FREQUENCY = DEFAULT_ROUTE_FREQUENCY
-                except Exception as e:
+                except ValueError as e:
```
Comment (nlehuby) [inline creators/accra/trips_creator_accra.py]: You are right, this is a bad pattern. But you also need to catch TypeError exception

File: creators/accra/trips_creator_accra.py (hunk 1 above)
Comment (grote) [reply creators/accra/trips_creator_accra.py]: Ups, thanks for noticing. Nasty copy and paste mistake I guess.

**Pull request merged**